docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3.4'
+services:
+    web_base: &web_base
+        build:
+            context: .
+            dockerfile: server/Dockerfile
+            target: development
+        command: [
+            "./node_modules/.bin/nodemon",
+            "--watch",
+            "lib/**/*",
+            "--watch",
+            "server/**/*",
+            "server/index.js"
+        ]
+        environment:
+            - NODE_ENV=development
```

Comment:
Does this take any effect?  The application itself (i.e. in the manifest) only pays attention to whether `NODE_ENV=production` or not.

Reply:
It might not functionally, but I believe that it is best practice to always define a `NODE_ENV` variable which is why I've added it here.